AI/Nullkiller/Pathfinding/Actors.cpp

```diff
@@ -461,15 +459,6 @@ CCreatureSet * DwellingActor::getDwellingCreatures(const CGDwelling * dwelling,
 			continue;
 
 		auto creature = creatureInfo.second.back().toCreature();
-		auto count = creatureInfo.first;
-
-		if(waitForGrowth)
```

Comment:
Probably it was discarded at certain stage because of complexity. At the very beginning hero chain was allowed to by army from dwellings. Now only from town.